run-tests.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -eEuo pipefail
+
+readonly POCKETIC_VERSION="8.0.0"
+readonly POCKETIC_URL="https://github.com/dfinity/pocketic/releases/download/${POCKETIC_VERSION}/pocket-ic-x86_64-linux.gz"
+readonly ASSET_WASM_URL="https://github.com/dfinity/sdk/raw/master/src/distributed/assetstorage.wasm.gz"
+readonly WORKDIR="$(pwd)"
+readonly CANISTER_DIR="${WORKDIR}/canister_wasms"
+readonly POCKETIC_BIN="${WORKDIR}/pocket-ic"
+readonly CARGO_TARGET_DIR="${WORKDIR}/target/debug"
+
+log() { echo "[$(date -u +%Y-%m-%dT%H:%M:%SZ)] $*" >&2; }
+
+log "Downloading PocketIC v${POCKETIC_VERSION}"
+curl -fsSL --retry 3 --retry-delay 5 "${POCKETIC_URL}" -o pocket-ic.gz || {
+  log "Failed to download PocketIC"
+  exit 1
+}
+log "Extracting PocketIC"
+gzip -df pocket-ic.gz || { log "Failed to extract PocketIC"; exit 1; }
+chmod +x "${POCKETIC_BIN}" || { log "Failed to make PocketIC executable"; exit 1; }
+export POCKET_IC_BIN="${POCKETIC_BIN}"
+log "PocketIC setup completed"
+
+log "Building ic-gateway"
+cargo build --verbose || { log "ic-gateway build failed"; exit 1; }
+export CARGO_TARGET_DIR
+log "ic-gateway build completed"
+
+log "Downloading asset canister WASM"
+mkdir -p "${CANISTER_DIR}" || { log "Failed to create canister directory"; exit 1; }
+curl -fsSL --retry 3 --retry-delay 5 "${ASSET_WASM_URL}" -o "${CANISTER_DIR}/assetstorage.wasm.gz" || {
```

Comment:
Also checksum would be good

Reply:
here we'll use a particular commit to download `assetstorage.wasm.gz` (and a corresponding `sha256sum`), canister wasm is not published as a release in the `sdk` repo